data/abilities/verification/0df4d46e-e202-4b29-9a19-c2540982002d.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Consider instead of adding these auto-loaded and then changing their access to HIDDEN after they're loaded, you might just be able to build then and store them at runtime (build the objects in Python and insert them into the data svc directly).

Reply:
Agreed. Sounds good 👍